addOns/client/src/main/java/org/zaproxy/addon/client/ExtensionClientIntegration.java

```diff
@@ -547,6 +569,31 @@ public void addReportedObject(ReportedObject obj) {
                                 LOGGER.error(e.getMessage(), e);
                             }
                         });
+        decPscanCount();
+    }
+
+    private void incPscanCount() {
+        if (hasView()) {
+            ThreadUtils.invokeLater(
+                    () -> {
+                        this.pscanStatus.incScanCount();
+                        LOGGER.debug(
```

Comment:
Do we really need these debugs? I can see why they'd be v useful while making this change, but once its done I think they will just add noise..